gatewayconfig/file_loader.py

```diff
@@ -6,10 +6,13 @@
 
 
 def read_eth0_mac_address(eth0_mac_address_filepath):
-    return open(eth0_mac_address_filepath) \
-            .readline() \
-            .strip() \
-            .upper()
+    try:
+        return open(eth0_mac_address_filepath) \
+                .readline() \
+                .strip() \
+                .upper()
+    except FileNotFoundError:
+        return "FF:FF:FF:FF:FF:FF"
```

Comment:
IIRC @vpetersson wanted to move away from the FF:FF style and throw useful error messages instead

Reply:
@shawaj Right now i do the same way what we using for wifi mac address:
https://github.com/NebraLtd/hm-config/blob/2f0125ca6d053e27f66748fb3119fd96b268df98/gatewayconfig/file_loader.py#L18-L25
If we want to rewrite this logic completely. I think this is now worthy of a separate ticket.
Correct me if I'm wrong.

Reply:
Yes @ilyastrodubtsev I agree this is best way for now and then to change in future ticket if @vpetersson thinks there is better way perhaps 👍